docs/devel/e2e-node-tests.md

```diff
@@ -63,35 +63,54 @@ and remote passwordless sudo access over ssh)
 * using [run_e2e.go](../../test/e2e_node/runner/run_e2e.go) to build a tar.gz
 and executing on host (requires host access w/ remote sudo)
 
-### Configuring a new remote host for testing
+### Option 1: Configuring a new remote host from scratch for testing
 
-The host must contain a environment capable of supporting a mini-kubernetes
-cluster. Includes:
+The host must contain an environment capable of running a minimal kubernetes cluster
+consisting of etcd, the kube-apiserver, and kubelet. The steps required to step a host vary between distributions
+(coreos, rhel, ubuntu, etc), but may include:
 * install etcd
 * install docker
+* add user running tests to docker group
 * install lxc and update grub commandline
 * enable tty-less sudo access
 
-See [setup_host.sh](../../test/e2e_node/environment/setup_host.sh)
+These steps should be captured in [setup_host.sh](../../test/e2e_node/environment/setup_host.sh)
+
+### Option 2: Copying an existing host image from another project
+
+If there is an existing image in another project you would like to use, you can use the script
+[copy-e2e-image.sh](../../test/e2e_node/jenkins/copy-e2e-image.sh) to copy an image
+from one GCE project to another.
+
+```sh
+copy-e2e-image.sh <image-to-be-copied-name> <from-gce-project> <to-gce-project>
+```
 
 ### Running the tests
 
-1. If running against a host on gce
+1. Make sure you have ginkgo installed on your PATH: `which ginkgo`.  If it is not installed, you can
```

Comment:
nit: Can't we do this automatically with Makefiles? Install ginkgo if it is not found.
Also why install ginkgo in user's gopath instead of inside the `_output` directory? AFAIK, `build-go.sh` outputs binaries into the `_output` directory.